### PR TITLE
[Linux] Update open-vm-tools test cases

### PR DESCRIPTION
**Pull request by keirazhang**
Changes in this pull request include:
1. Renamed `ovt_verify_install` to `ovt_verify_pkg_install`
2. Renamed `ovt_verify_uninstall` to `ovt_verify_pkg_uninstall`
3. Added a new variable `vmtools_install_type` to identify how open-vm-tools is installed: package or unknown
4. Skipped OVT test cases when `vmtools_install_type` is not `package`
5. Removed some meaningless conditions or variables
6. Initialized VGAuthService related variables

### CODE REVIEW

File: linux/open_vm_tools/ovt_verify_pkg_install.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Copyright 2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+#   This test case is to verify open-vm-tools installation with guest OS package manager.
+# If guest OS doesn't have open-vm-tools, this case will install open-vm-tools from OS ISO image
+# (e.g. RHEL, SLES, SLED, etc),or install from official online repository (e.g. Ubuntu, Debian,
+# VMware Photon OS, CentOS, etc).
+# If it already has open-vm-tools installed from package, and update_tools parameter is set "true"
```
Comment (Tomorrow9) [inline linux/open_vm_tools/ovt_verify_pkg_install.yml:9]: update_vmtools

File: linux/open_vm_tools/ovt_verify_pkg_install.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+- name: ovt_verify_pkg_install
+  hosts: localhost
+  gather_facts: false
+  tasks:
+    - name: "Test case block"
+      block:
+        - name: "Test setup"
+          include_tasks: ../setup/test_setup.yml
+
+        - name: "Set update VMware tools by default"
```
Comment (Tomorrow9) [inline linux/open_vm_tools/ovt_verify_pkg_install.yml:23]: VMware Tools

File: linux/open_vm_tools/ovt_verify_pkg_install.yml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+#   This test case is to verify open-vm-tools installation with guest OS package manager.
+# If guest OS doesn't have open-vm-tools, this case will install open-vm-tools from OS ISO image
+# (e.g. RHEL, SLES, SLED, etc),or install from official online repository (e.g. Ubuntu, Debian,
+# VMware Photon OS, CentOS, etc).
+# If it already has open-vm-tools installed from package, and update_tools parameter is set "true"
+# in vars/test.yml, it will reinstall open-vm-tools.
+# If it already has open-vm-tools not installed from package, this test case will be skipped with
```
Comment (Tomorrow9) [inline linux/open_vm_tools/ovt_verify_pkg_install.yml:11]: This description "If it already has open-vm-tools not installed from package" seems need to be refined.

File: linux/open_vm_tools/ovt_verify_pkg_uninstall.yml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+            skip_msg: "Skip test case because {{ guest_os_ansible_distribution }} doesn't support uninstalling open-vm-tools"
+            skip_reason: "Not Supported"
+          when: guest_os_ansible_distribution in ['Flatcar', 'RHCOS']
+
+        - name: "Block test case when guest OS doesn't have open-vm-tools"
+          include_tasks: ../../common/skip_test_case.yml
+          vars:
+            skip_msg: "Test case '{{ ansible_play_name }}' is blocked because guest OS doesn't have open-vm-tools installed."
+            skip_reason: "Blocked"
+          when: vmtools_is_installed is undefined or not (vmtools_is_installed | bool)
```
Comment (Tomorrow9) [inline linux/open_vm_tools/ovt_verify_pkg_uninstall.yml:35]: vmtools_is_installed is undefined can also be removed.

File: linux/open_vm_tools/ovt_verify_pkg_install.yml (hunk 1 above)
Comment (keirazhang) [reply linux/open_vm_tools/ovt_verify_pkg_install.yml]: Nice catch. Fixed. Thanks!

File: linux/open_vm_tools/ovt_verify_pkg_install.yml (hunk 2 above)
Comment (keirazhang) [reply linux/open_vm_tools/ovt_verify_pkg_install.yml]: Fixed.

File: linux/open_vm_tools/ovt_verify_pkg_install.yml (hunk 3 above)
Comment (keirazhang) [reply linux/open_vm_tools/ovt_verify_pkg_install.yml]: Updated.

File: linux/open_vm_tools/ovt_verify_pkg_uninstall.yml (hunk 4 above)
Comment (keirazhang) [reply linux/open_vm_tools/ovt_verify_pkg_uninstall.yml]: Removed.

**Review by Tomorrow9 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**